packages/devtools_app/lib/src/debugger/flutter/debugger_controller.dart

```diff
@@ -41,6 +44,12 @@ class DebuggerController extends DisposableController
 
   ValueListenable<bool> get isPaused => _isPaused;
 
+  final _maybeResuming = ValueNotifier<bool>(false);
+
+  /// This indicates that we've requested a resume (or step) operation from the
+  /// VM, but haven't yet received the 'resumed' isolate event.
+  ValueListenable<bool> get maybeResuming => _maybeResuming;
```

Comment:
why is this maybeResuming instead of just resuming? "maybe" makes it seem like we can never really guarantee that this value is the true value of "resuming"

Reply:
I think I didn't know that this would be that precise when I wrote this; I'll change to something like `resuming` or `resumeInProgress`.